src/components/dashboard/UpcomingTrips.js

```diff
@@ -23,6 +33,13 @@ export default class UpcomingTrips extends Component {
           <p>{trip.name}</p>
         </Link>
         <p>{trip.dscript}</p>
+        <Button
+          variant={'danger'}
+          className={'float-right'}
+          onClick={() => this.handleDelete(trip)}
```

Comment:
💣💣💣

Side note - thoughts on abstracting this delete button into its own class? We could then reuse it for all of our delete functionality. My thought is that we could place this in our `global` folder just like the return, submit, and edit button. (Doesn't need to be done in this pull request, of course).

Reply:
Hmm, this might be a good idea if we get a chance